### PR TITLE
[libsystemd] add libcrypt as dependency - more backports

### PR DESCRIPTION
**Pull request by Tobulus**
Specify library name and version:  **libsystemd**

Followup for [this PR](https://github.com/conan-io/conan-center-index/pull/18497). There are more backports released in the meantime.

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by Tobulus** [comment]
@RubenRBS triggering as this is a follow-up

**Comment by RubenRBS** [comment]
Pinging @jcar87, as I'll be out of the office for a bit :)

**Comment by Tobulus** [comment]
@RubenRBS quite some time passed. Pinging again.

**Comment by Tobulus** [comment]
@RubenRBS @franramirez688 @jcar87 
I don't get the way/order how pull requests are processed. I know that you guys are busy in various ways in the conan client or conan center context as stated in some other context.

But: I can see pull requests merged every day which don't have a bigger scope than e.g. this one and where opened in the same week of merging.

This is completley frustrating for everyone who voluntarily provides improvements to recipes. And I'm not even talking about other stuff is reviewed faster: this is creating extra work in form of merge conflicts. There is even a bot which provides the Info that another PR is doing changes to the same recipe which should bei considered...

Current example: https://github.com/conan-io/conan-center-index/pull/20557

Why is there no general FIFO order for reviews? Of course there will be exceptions. But in general this could be done.

**Review by danimtb — DISMISSED** [review]

**Review by jwillikers — APPROVED** [review]

File: recipes/libsystemd/all/conandata.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ patches:
       patch_description: "allow to build in environments without 'realpath --relative-to' by replacing it with conan-specific build variable"
       patch_type: "conan"
   "253.14":
-    - patch_file: "patches/253.3/0001-missing_syscalls.py-Replace-unicode-with-ascii.patch"
+    - patch_file: "patches/253.6/0001-missing_syscalls.py-Replace-unicode-with-ascii.patch"
       patch_description: "allow to use meson.build with older versions of Python by replacing utf8 message to ascii message in the helper script"
       patch_type: "conan"
     - patch_file: "patches/251.15/0001-Remove-dependency-from-coreutils.patch"
```
Comment (danimtb) [inline recipes/libsystemd/all/conandata.yml:48]:
```suggestion
    - patch_file: "patches/251.18/0001-Remove-dependency-from-coreutils.patch"
```

File: recipes/libsystemd/all/conandata.yml [hunk 2]
```diff
@@ -53,7 +53,7 @@ patches:
       patch_description: "fixes cross-compilation by passing necessary compiler arguments to generator scripts"
       patch_type: "portability"
   "253.10":
-    - patch_file: "patches/253.3/0001-missing_syscalls.py-Replace-unicode-with-ascii.patch"
+    - patch_file: "patches/253.6/0001-missing_syscalls.py-Replace-unicode-with-ascii.patch"
       patch_description: "allow to use meson.build with older versions of Python by replacing utf8 message to ascii message in the helper script"
       patch_type: "conan"
     - patch_file: "patches/251.15/0001-Remove-dependency-from-coreutils.patch"
```
Comment (danimtb) [inline recipes/libsystemd/all/conandata.yml:59]:
```suggestion
    - patch_file: "patches/251.18/0001-Remove-dependency-from-coreutils.patch"
```

File: recipes/libsystemd/all/conandata.yml [hunk 3]
```diff
@@ -64,7 +64,7 @@ patches:
       patch_description: "fixes cross-compilation by passing necessary compiler arguments to generator scripts"
       patch_type: "portability"
   "253.6":
-    - patch_file: "patches/253.3/0001-missing_syscalls.py-Replace-unicode-with-ascii.patch"
+    - patch_file: "patches/253.6/0001-missing_syscalls.py-Replace-unicode-with-ascii.patch"
       patch_description: "allow to use meson.build with older versions of Python by replacing utf8 message to ascii message in the helper script"
       patch_type: "conan"
     - patch_file: "patches/251.15/0001-Remove-dependency-from-coreutils.patch"
```
Comment (danimtb) [inline recipes/libsystemd/all/conandata.yml:70]:
```suggestion
    - patch_file: "patches/251.18/0001-Remove-dependency-from-coreutils.patch"
```

**Review by danimtb — APPROVED** [review]

**Review by RubenRBS — APPROVED** [review]

**Pull request merged**